integral_view/templates/view_zfs_pool.html

```diff
@@ -82,22 +82,30 @@ <h5 class="panel-title">
-                  {{pool.usage.capacity.value|floatformat}}%
+                <div class="progress-bar {% if pool.usage.used_percent > 50 and pool.usage.used_percent < 70 %}  progress-bar-warning {% elif pool.usage.used_percent >= 70 %} progress-bar-danger {% else %} progress-bar-info {%endif%}" role="progressbar" aria-valuenow="{{pool.usage.used_percent}}" aria-valuemin="0" aria-valuemax="100.0" style="{%if pool.usage.used_percent %} width: {{pool.usage.used_percent}}%;{%endif%}min-width: 3em;">
+                  {{pool.usage.used_percent}}%
                 </div>
               </div>
               <table class="table table-bordered table-hover table-responsive" style="width:800px">
-                <tr><th> Space available for data </th> <td>{{pool.usage.available_for_data.value}}{{pool.usage.available_for_data.unit}}<td></tr>
+                <tr><th> Total consumable space {% if pool.properties.quota.value != 'none' %}* {% endif %}</th> <td>{{pool.usage.total_space}}<td></tr>
                 <tr><th> Space consumed by data </th> <td>{{pool.usage.used_by_data.value}}{{pool.usage.used_by_data.unit}}<td></tr>
+                <tr><th> Space available for data </th> <td>{{pool.usage.available_for_data.value}}{{pool.usage.available_for_data.unit}}<td></tr>
+		<!--
                 <tr><th> Raw space from all data disks </th> <td>{{pool.usage.size.value}}{{pool.usage.size.unit}}<td></tr>
                 <tr><th> Space consumed by data & metadata </th> <td>{{pool.usage.allocated.value}}{{pool.usage.allocated.unit}}<td></tr>
                 <tr><th> Space consumed percentage </th> <td>{{pool.usage.capacity.value}}{{pool.usage.capacity.unit}}<td></tr>
+                <tr><th> Consumed space percentage </th> <td>{{pool.usage.used_percent}}%<td></tr>
+		-->
+                <tr><th> Percentage of space available </th> <td>{{pool.usage.avail_percent}}%<td></tr>
                 <tr><th> Pool dedup </th> <td>{{pool.usage.dedup}}<td></tr>
                 {%if pool.usage.dedup == 'on'%}
                   <tr><th> Pool dedup ratio</th> <td>{{pool.usage.dedupratio.value}}{{pool.usage.dedupratio.unit}}<td></tr>
                 {%endif%}
                 <tr><th> Pool fragmentation percentage</th> <td>{{pool.usage.fragmentation.value}}{{pool.usage.fragmentation.unit}}<td></tr>
                 <tr><th> Pool autoexpand </th> <td>{%if pool.usage.autoexpand%} Enabled {%else%} Disabled{%endif%}<td></tr>
+		{% if pool.properties.quota.value != 'none' %}
+                  <tr><td>* This is the storage space after reserving 15% for optimal system operation.</td></tr>
```

Comment:
Hey, this may not always be 15%! Just realized that this can be changed now from the UI. So we should make this a more generic comment.